swap_meet/vendor.py

```diff
@@ -1,2 +1,59 @@
+
+import uuid
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
+        self.inventory = [] if inventory is None else inventory
```

Comment:
Nice job setting the default value of inventory to None since default parameters should not be initialized to empty mutable datatypes!